stable/pomerium/templates/secret.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: {{ template "pomerium.name" . }}
+    chart: {{ template "pomerium.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "pomerium.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  cookie-secret: {{ .Values.config.cookieSecret | b64enc }}
+  shared-secret: {{ .Values.config.sharedSecret | b64enc }}
+  idp-client-id: {{ .Values.authenticate.idp.clientID | b64enc }}
+  idp-client-secret: {{ .Values.authenticate.idp.clientSecret | b64enc }}
+
+{{- if .Values.tls.generateTLS }}
+{{- $ca := genCA "default-ca" 365 }}
+{{- $cn := default "example.com" .Values.tls.defaultCN }}
+  ca-cert: {{ $ca.Cert | b64enc | b64enc }}
+  ca-key: {{ $ca.Key | b64enc | b64enc }}
```

Comment:
@desimone this is not used anywhere

Reply:
I'm not sure what best practice would be for setting up a dummy ca. Would you expect to be able to recover the key or would be fine with throwing it away? Of course the ideal case is to use "real" pki infrastructure but for one-off here I thought it may be better to keep it ? (to revoke?)

Reply:
I didn't even think of that ^^ I just noticed that it wasn't used and thought that since it was not for prod, it wasn't very important. So that's a good question actually, but I have no answer :)

Reply:
I think it is fine to keep for a user generating a dummy ca, and gives them the flexibility to revoke, or extend signed certs later. But I should sufficiently warn users in the docs.